tests/scripts/create-dev-cluster.sh

```diff
@@ -141,6 +141,13 @@ show_usage() {
 ####################################################################
 ################# MAIN #############################################
 
+
+invocation_error() {
```

Comment:
Is this function part of 
`################# MAIN #############################################` ?